dashboard/src/components/AppView/ChartInfo.test.tsx

```diff
@@ -1,27 +1,54 @@
 import { shallow } from "enzyme";
+import context from "jest-plugin-context";
 import * as React from "react";
 
+import { Link } from "react-router-dom";
 import { hapi } from "shared/hapi/release";
 import ChartInfo from "./ChartInfo";
 
+const defaultProps = {
+  app: {
+    chart: {
+      metadata: {
+        appVersion: "0.0.1",
+        description: "test chart",
+        icon: "icon.png",
+        version: "1.0.0",
+      },
+    },
+    name: "foo",
+  } as hapi.release.Release,
+  updateCheck: { checked: false },
```

Comment:
I guess this should be `undefined` and also be `updateInfo`? Could also just drop it.

Reply:
true, I missed this one